### PR TITLE
New leaderboard for API performance/cost/speed, release today

### PR DESCRIPTION
**Pull request by clefourrier**

### CODE REVIEW

File: leaderboard-artificial-analysis.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+We test every API endpoint on the leaderboard 8 times per day, and leaderboard figures represent the median measurement of the last 14 days. We also have percentile breakdowns within the collapsed tabs.
+
+Quality metrics are currently collected on a per-model basis and show results reports by model creators, but watch this space as we begin to share results from our independent quality evaluations across each endpoint. 
+
+For further definitions, see our [full methodology page](https://artificialanalysis.ai/methodology).
+
+## Highlights (May 2024, see the leaderboard for the latest)
+
+- The language models market has exploded in complexity over the last year. Launches that have shaken up the market just within the last two months include proprietary models like Anthropic's Claude 3 series and open models such as Databricks' DBRX, Mistral's Mixtral 8x22B and Meta's Llama 3.
+- Price and speed vary considerably between models and providers. From Claude 3 Opus to Llama 3 8B, there is a 300x pricing spread - that's more than two orders of magnitude!
```
Comment (NathanHB) [inline leaderboard-artificial-analysis.md:64]: llama-3 and more generally open source models can also be run locally and finetnued to fit the user's need at low cost

File: leaderboard-artificial-analysis.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+The leaderboard allows exploration of performance under several different workloads (6 combinations in total):
+
+- varying the **prompt length**: ~100 tokens, ~1k tokens, ~10k tokens.
+- running **parallel queries**: 1 query, 10 parallel queries.
+
+### Methodology
+
+We test every API endpoint on the leaderboard 8 times per day, and leaderboard figures represent the median measurement of the last 14 days. We also have percentile breakdowns within the collapsed tabs.
+
+Quality metrics are currently collected on a per-model basis and show results reports by model creators, but watch this space as we begin to share results from our independent quality evaluations across each endpoint.
```
Comment (NathanHB) [inline leaderboard-artificial-analysis.md:57]: interesting, how are they going to eval closed source models ?

File: leaderboard-artificial-analysis.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+![Untitled](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/leaderboards-on-the-hub/artificialanalysis_leaderboard-may.jpg)
+
+Source: [LLM Performance Leaderboard](https://huggingface.co/spaces/ArtificialAnalysis/LLM-Performance-Leaderboard)
+
+### Metric coverage
+
+The metrics reported are:
+
+- **Quality:** a simplified index for comparing model quality and accuracy, calculated based on metrics such as MMLU, MT-Bench and Chatbot Arena ranking.
```
Comment (NathanHB) [inline leaderboard-artificial-analysis.md:38]: Those are the ones reported by the companies / labs making the models right ?

Comment (clefourrier) [reply leaderboard-artificial-analysis.md:38]: Interesting - there is indeed an ambiguity, I believe the metrics are pulled from the leaderboards

File: leaderboard-artificial-analysis.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
```
Comment (clefourrier) [inline leaderboard-artificial-analysis.md:38]:
```suggestion
- **Quality:** a simplified index for comparing model quality and accuracy, calculated based on metrics such as MMLU, MT-Bench, HumanEval scores, as reported by the model authors, and Chatbot Arena ranking.
```

**Pull request merged**